packages/arb-util/hashing/hashing.go

```diff
@@ -58,26 +66,46 @@ func TimeBlocks(input *common.TimeBlocks) []byte {
 	return solsha3.Uint128(input.AsInt())
 }
 
+// Uint256 converts input to its packed ABI encoding
 func Uint256(input *big.Int) []byte {
 	return solsha3.Uint256(new(big.Int).Set(input))
 }
 
+// Uint128 converts input to its packed ABI encoding
 func Uint128(input *big.Int) []byte {
 	return solsha3.Uint128(new(big.Int).Set(input))
 }
 
+// Uint256Array converts input to its packed ABI encoding
+func Uint256Array(input []*big.Int) []byte {
+	ints := make([]*big.Int, 0, len(input))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)